cmd/allocator/main.go

```diff
@@ -306,6 +309,9 @@ func main() {
 		}
 	}
 
+	err = http.ListenAndServe(":8080", http.DefaultServeMux)
+	logger.WithError(err).Fatal("allocation service crashed")
+
```

Comment:
If I move these two lines to happen at the end of the function, /metrics stops working. But if I leave it here, it will shadow lines 316-328, basically everything below it.